elastic/kibana
# [SecuritySolution] Update file validation because the file type is empty on windows

## Summary

Bug: On Windows, we can't select TSV files or drag them to the file picker. This error happens because we have a validation for the file type, and for some reason, on Windows, it is an empty string 🤡 .

I updated the file validation logic to allow empty strings for file types and added one extra value to the acceptable types, `.tsv`. So, when the mime type checks fail, it will accept any files with the `tsv` extension.


I desk-tested it on Windows-edge, Windows-chrome, Mac-chrome and it works.

### How to test it?
* ON A WINDOWS MACHINE
* Open the security solution app and navigate to the entity store page
* Drag a TSV file created on a Windows machine; the file should be accepted
* Click on the "select file" button and select a TSV file created on a Windows machine
* Both methods should allow you to select the file, and you should be able to proceed with the bulk upload

#### Do I need to checkout your code and start kibana?
If you are a lazy person like me, you can test the feature using the cloud deployment [here](https://kibana-pr-199791.kb.us-west2.gcp.elastic-cloud.com/app/security/entity_analytics_entity_store).
You can get the login/password using Vault. The instructions are on the latest Cloud Deployment link, or ask me for credentials.

#### What if I don't have a Windows machine?
Buy one. I am kidding. 🤡 
You can follow [this tutorial](https://github.com/elastic/security-team/blob/main/tools/sec-eng-prod/sep-vms/README.md#setup) to deploy it to gcloud, and connect to it using RDP.
But If you are lazy like me, you can ask me for my Windows 11 cloud deployment credentials.

_Obs. You need an app to access the cloud deployment via RDP. The "Windows App" can be easily downloaded from the App Store on Mac._

### Checklist

Delete any items that are not applicable to this PR.
